x-pack/plugins/osquery/public/actions/actions_table.tsx

```diff
@@ -131,9 +134,22 @@ const ActionsTableComponent = () => {
     },
     [push]
   );
+
+  const existingPackIds = useMemo(() => map(packsData?.data ?? [], 'id'), [packsData]);
```

Comment:
why ?? []?

Reply:
in case the `packsData` wasn't loaded yet